test/functional/sqlsrv/test_non_alpha_password.phpt

```diff
@@ -51,5 +62,5 @@ print "Test successful";
 ?> 
 --EXPECTREGEX--
 An unescaped right brace \(}\) was found in either the user name or password.  All right braces must be escaped with another right brace \(}}\)\.
-Warning: sqlsrv_close\(\) expects parameter 1 to be resource, boolean given in .+(\/|\\)test_non_alpha_password\.php on line 38
+Warning: sqlsrv_close\(\) expects parameter 1 to be resource, boolean given in .+(\/|\\)test_non_alpha_password\.php on line 50
```

Comment:
Even so I think it would be better to put in a message in the main body of the test - `if ($conn) {sqlsrv_close($conn); echo "Error, $conn should not be valid\n";}` instead of using PHP's standard error message here.